src/index.js

```diff
@@ -268,7 +268,8 @@ class Toast extends Component {
           'text1',
           'text2',
           'hide',
-          'show'
+          'show',
+          'customOptions'
```

Comment:
Hi @troyvnit , thanks for taking the time to do this!

Can we rename it to `props`? I think it would come naturally to everyone using it

Reply:
Sure, I changed. Thanks!